aisynphys/ui/notebook.py

```diff
@@ -196,13 +211,15 @@ def show_connectivity_matrix(ax, results, pre_cell_classes, post_cell_classes, c
             cprob[i,j] = cp
             if ctype == 'chemical':
                 cprob_str[i,j] = "" if result['n_probed'] == 0 else "%d/%d" % (found, result['n_probed'])
+                if (show_pmax or correction_only) and np.isfinite(cp):
+                    cprob_str[i,j] += "\n %.3f" %(cp)
             elif ctype == 'electrical':
                 cprob_str[i,j] = "" if result['n_gaps_probed'] == 0 else "%d/%d" % (found, result['n_gaps_probed'])
             cprob_alpha[i,j] = 1.0 - 2.0 * max(cp_upper_ci - cp, cp - cp_lower_ci)
 
     # map connection probability to RGB colors
     mapper = matplotlib.cm.ScalarMappable(norm=norm, cmap=cmap)
-    cprob_rgba = mapper.to_rgba(np.clip(cprob, 0.01, 1.0))
+    cprob_rgba = mapper.to_rgba(np.clip(cprob, norm.vmin, norm.vmax))
```

Comment:
Maybe we need to make this configurable--there are cases where we want a consistent color range that does not change depending on the data range.

Reply:
Perhaps I confused you in the commit notes.
I meant, 'RGBA map now respects the range the data are clipped'.
This 'norm' variable can be set outside the function (like below) to make the range consistent.

# define a colormap and log normalization used to color the heatmap
norm = matplotlib.colors.LogNorm(vmin=0.01, vmax=1.0, clip=True)
cmap = matplotlib.cm.get_cmap('plasma')

So, as long as you keep this argument the same, the color range would be consistent across figures.